### PR TITLE
[zh] translate concepts/API-initiated Eviction

### LINKED ISSUE #28156: [zh] translate concepts/API-initiated Eviction
**This is a Feature Request**
translate /docs/concepts/scheduling-eviction/api-eviction.md into Chinese

**What would you like to be added**
/zh/docs/concepts/scheduling-eviction/api-eviction.md

/language zh
/assign

### PR DESCRIPTION
**Pull request by zhiguo-lu**
translate /docs/concepts/scheduling-eviction/api-eviction.md into Chinese

close #28156

### CODE REVIEW

File: content/zh/docs/concepts/scheduling-eviction/api-eviction.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<!-- 
+You can request eviction by directly calling the Eviction API 
+using a client of the kube-apiserver, like the `kubectl drain` command. 
+This creates an `Eviction` object, which causes the API server to terminate the Pod. 
+
+API-initiated evictions respect your configured [`PodDisruptionBudgets`](/docs/tasks/run-application/configure-pdb/)
+and [`terminationGracePeriodSeconds`](/docs/concepts/workloads/pods/pod-lifecycle#pod-termination). 
+-->
+你可以通过 kube-apiserver 的客户端，比如 `kubectl drain` 这样的命令，直接调用驱逐 API 发起驱逐。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/api-eviction.md:23]:
```suggestion
你可以通过 kube-apiserver 的客户端，比如 `kubectl drain` 这样的命令，直接调用 Eviction API 发起驱逐。
```

File: content/zh/docs/concepts/scheduling-eviction/api-eviction.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+此操作创建一个 `Eviction` 对象，该对象再驱动 API 服务器终止选定的Pod。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/api-eviction.md:24]:
```suggestion
此操作创建一个 `Eviction` 对象，该对象再驱动 API 服务器终止选定的 Pod。
```

File: content/zh/docs/concepts/scheduling-eviction/api-eviction.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+API 发起的驱逐将依据你的
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/api-eviction.md:26]:
```suggestion
API 发起的驱逐将遵从你的
```

File: content/zh/docs/concepts/scheduling-eviction/api-eviction.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+[`PodDisruptionBudgets`](/zh/docs/tasks/run-application/configure-pdb/)
+和 [`terminationGracePeriodSeconds`](/zh/docs/concepts/workloads/pods/pod-lifecycle#pod-termination)
+配置为准。
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/api-eviction.md:29]:
```suggestion
配置。
```

File: content/zh/docs/concepts/scheduling-eviction/api-eviction.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## {{% heading "whatsnext" %}}
+
+<!-- 
+* Learn about [Node-pressure Eviction](/docs/concepts/scheduling-eviction/node-pressure-eviction/)
+* Learn about [Pod Priority and Preemption](/docs/concepts/scheduling-eviction/pod-priority-preemption/)
+-->
+* 了解 [节点压力引发的驱逐](/zh/docs/concepts/scheduling-eviction/node-pressure-eviction/)
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/api-eviction.md:37]:
```suggestion
* 了解[节点压力引发的驱逐](/zh/docs/concepts/scheduling-eviction/node-pressure-eviction/)
```

File: content/zh/docs/reference/glossary/api-eviction.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+---
+title: API 发起的驱逐
+id: api-eviction
+date: 2021-04-27
+full_link: /zh/docs/concepts/scheduling-eviction/pod-eviction/#api-eviction
+short_description: >
+  API 发起的驱逐是一个先调用驱逐 API 创建驱逐对象，再由该对象优雅的中止 Pod 的过程。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/api-eviction.md:7]:
```suggestion
  API 发起的驱逐是一个先调用驱逐 API 创建 Eviction 对象，再由该对象体面地中止 Pod 的过程。
```

File: content/zh/docs/reference/glossary/api-eviction.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+- operation
+---
+-->
+<!-- 
+API-initiated eviction is the process by which you use the [Eviction API](/docs/reference/generated/kubernetes-api/{{<param "version">}}/#create-eviction-pod-v1-core)
+to create an `Eviction` object that triggers graceful pod termination.
+-->
+API 发起的驱逐是一个先调用
+[驱逐 API](/zh/docs/reference/generated/kubernetes-api/{{<param "version">}}/create-eviction-pod-v1-core)
+创建驱逐对象，再由该对象优雅的中止 Pod 的过程。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/api-eviction.md:32]:
```suggestion
创建 Eviction 对象，再由该对象体面地中止 Pod 的过程。
```

File: content/zh/docs/reference/glossary/api-eviction.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+tags:
+- operation
+---
+-->
+<!-- 
+API-initiated eviction is the process by which you use the [Eviction API](/docs/reference/generated/kubernetes-api/{{<param "version">}}/#create-eviction-pod-v1-core)
+to create an `Eviction` object that triggers graceful pod termination.
+-->
+API 发起的驱逐是一个先调用
+[驱逐 API](/zh/docs/reference/generated/kubernetes-api/{{<param "version">}}/create-eviction-pod-v1-core)
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/api-eviction.md:31]:
```suggestion
[Eviction API](/docs/reference/generated/kubernetes-api/{{<param "version">}}/create-eviction-pod-v1-core)
```

The API reference doesn't have a localized version.

File: content/zh/docs/reference/glossary/api-eviction.md (hunk 7 above)
Comment (zhiguo-lu) [reply content/zh/docs/reference/glossary/api-eviction.md]: 的->地，细节！

**Review by tanjunchen — COMMENTED** [review]
/lgtm

**Pull request merged**